src/handlers/auth.js

```
@@ -8,19 +8,18 @@ import { health } from "./health";
 
 /**
  * @type {Record<string,(routerClient: RouterClient) => Promise<void>>}
  */
 const routeMap = {
-  create_org: createOrg,
-  register,
-  setup,
-  login,
-  logout,
-  health,
+  [process.env.KINDE_AUTH_CREATEORG_ROUTE || 'create_org']: createOrg,
```

ESLint

[error] 'process' is not defined.

(no-undef)